projects/valtimo/access-control/src/lib/models/permission.types.ts

```diff
@@ -0,0 +1,11 @@
+enum PERMISSION_ACTION {
```

Comment:
The enums don't seem like a good idea to me. Implementations might want to make use of actions/resources we have not programmed beforehand in the libs